### ISSUE TITLE
Nevanax EMP & Meltdowns.

### ISSUE THREAD

**Issue by Ineedause** [Anuken/Mindustry-Suggestions#2650]
**Describe what you would like changed, and why.**

The EMP of the Navanax right now explodes when it's bullet life time ends or when it hits a enemy unit, this should not be the case. Like the Toxopid frag cannon, it should explode where your mouse is, this is for multiple reasons. 
1. so that you can actually use the healing property of the EMP
2. so that you can place the AOE affect more efficiently. 

On top of this change, the EMP needs a damage buff. As of right now there is no reason to go for Nevanax, you might as well go for Omura because of the longer range and Omura wins a 1 v 1. An EMP buff is needed to make the Nevadax useful. With that the Meltdowns need a range buff. Of all the testing I have done with the Nevanax I have never been able to get all 4 meltdowns to attack one unit at one time essentially halting Navanax's firepower. 

To sum it up I propose:

1. Make the EMP detonation occur at your mouse pointer
2. Buff the EMP damage by a significant amount.
3. Buff the Meltdown range by a significant amount.

The Nevanax right now is in between a support unit and an attack unit, unfortunately this makes it almost useless because there are other units that get the job done better. Small changes like these will prevent the Nevanax from becoming the next Eclipse in terms of usability. 

**Describe the changes you want to propose. Include possible alternatives.**

In terms of this there are no real alternatives, 2 of these 3 changes in my opinion would help a lot while still keeping the unit fairly balanced. 

(let the comment war-zone begin)

**Comment by vinhsad** [comment]
1.wtf is Nevanax
2.Navanax meltdown beam is already the dps of unit
3.EMP is used to debuff/buffing alsi it deal aoe dmg
4.Navanax is better than Eclipse and a lot of other T5 (1 single navnax can't kill 1 omura, but 10 navanax can kill 10 omura)

**Comment by Bottinator22** [comment]
Navanax isn't meant to be a primary attacking unit, it's meant to be a support unit, helping out by destroying the enemy power lines and debuffing enemy units.

**Comment by Kyllingene** [comment]
Oh no! A unit with a more subtle use-case than "deal big damage"! Run!

**Comment by Ineedause** [comment]
Maybe you should actually read the suggestion and then talk. Last time I checked there was a part of the EMP being more useable, READ.

**Comment by vinhsad** [comment]
> Maybe you should actually read the suggestion and then talk. Last time I checked there was a part of the EMP being more useable, READ.
i'm talking about this:

> On top of this change, the EMP needs a damage buff. As of right now there is no reason to go for Nevanax, you might as well go for Omura because of the longer range and Omura wins a 1 v 1. An EMP buff is needed to make the Nevadax useful. With that the Meltdowns need a range buff. Of all the testing I have done with the Nevanax I have never been able to get all 4 meltdowns to attack one unit at one time essentially halting Navanax's firepower.
not your EMP target things

**Comment by Ineedause** [comment]
I understand, but from my personal experience nevanax underperforms in every situation I have put it in. For example, I have had 10 Nevanax go against 10 Omura and Omura still one.

**Comment by IIINSTINCT** [comment]
Huh
navanax can defeat all t5 units 
it has  electrified status effect that making units speed into 70% and reload into 60% 
4 lasers that deal 312 damage per sec and it is auto attack 
Emp cannon splash damage have arcs like attacks and priorities power nodes and can only countered by plast walls
Also its a SUPPORT UNIT why you want it to have more damage 
A naval unit that can build
Have 16% armor 

And if you want the emp cannons to be fully controllable 
Think first why Anuke didn't made it like toxopid cannons in the first place?

**Comment by Ineedause** [comment]
I must mention that there was almost no competition

**Comment by Ineedause** [comment]
4 lasers that deal 312 damage per sec and it is auto attack

First thing, only 2 lasers attack at one time hence the range buff.

And if you want the emp cannons to be fully controllable

There are 2 ways to look at this, from my perspective I cannot use the EMP's abilities without full control of the detonation area.

navanax can defeat all t5 units

And there is no way in hell this is true

**Comment by vinhsad** [comment]
![Screenshot_86](https://user-images.githubusercontent.com/43229798/124207108-f87e8b80-db0e-11eb-9181-c82b541da4d9.png)
![Screenshot_85](https://user-images.githubusercontent.com/43229798/124207113-fb797c00-db0e-11eb-953f-23c164a84654.png)

**Comment by Ineedause** [comment]
right, imma do some testing, see you in 30 mins. Will tell you the results

**Comment by Ineedause** [comment]
"All Navanaxs are AI controlled, other units are controlled by me"

Eclipse - Navanax: Navanax
Reign - Navanax: Reign
Toxopid - Nevanax: Toxopid
(short range) Corveus - Nevanax: Nevanax
(long range) Corverus - Nevanax: Corveus
(Short range) Omura - Nevanax: Omura 
(Long range) Omura - Nevanax: Omura
Sei - Nevanax: Home team advantage matters
Oct survive time: Indefinite because the EMPs don't deal enough damage to break the shield and Meltdowns are mitigated by it.

**Comment by vinhsad** [comment]
![Screenshot_104](https://user-images.githubusercontent.com/43229798/124210214-3c748f00-db15-11eb-8013-1efa8b21b127.png)
![Screenshot_103](https://user-images.githubusercontent.com/43229798/124210222-40081600-db15-11eb-8d48-12dd0fc1c895.png)
hmm...?

**Comment by vinhsad** [comment]
![Screenshot_108](https://user-images.githubusercontent.com/43229798/124210532-c6bcf300-db15-11eb-86d2-0421eb9612d4.png)
![Screenshot_107](https://user-images.githubusercontent.com/43229798/124210540-c91f4d00-db15-11eb-82e7-33d5922a76ab.png)
hmm...
(10v10)

**Comment by oinkymoo** [comment]
> Sei - Nevanax: Home team advantage matters

what does this mean

**Comment by Kyllingene** [comment]
It means that they were never made for 1v1's, rather, they do exceptionally well when paired with other units.

**Comment by ShadeStrike62** [comment]
> Oh no! A unit with a more subtle use-case than "deal big damage"! Run!

that basically sums up most suggestions, thanks, I love this comment...

**Comment by Bottinator22** [comment]
Navanaxes aren't designed to beat up other units more effectively, they're designed to assist in attacking, destroying power lines behind base walls and debuffing enemies. Pair them with other stuff.
